sli/fdstreamtest.cc

```diff
@@ -67,7 +67,8 @@ main()
   }
 
   o << "Line 1" << std::endl;
-  o << "Line 2" << std::endl << "Line 3" << std::endl;
+  o << "Line 2" << std::endl
+    << "Line 3" << std::endl;
```

Comment:
This file has nothing to do with the pull request and just got in due to clang reformatting. Please remove from PR.